homeassistant/components/hue/v2/sensor.py

```diff
@@ -122,14 +117,13 @@ def native_value(self) -> int:
         # scale used because the human eye adjusts to light levels and small
         # changes at low lux levels are more noticeable than at high lux
         # levels.
-        return int(10 ** ((self.resource.light.light_level - 1) / 10000))
+        return int(10 ** ((self.resource.light.value - 1) / 10000))
 
     @property
     def extra_state_attributes(self) -> dict[str, Any]:
         """Return the optional state attributes."""
         return {
-            "light_level": self.resource.light.light_level,
-            "light_level_valid": self.resource.light.light_level_valid,
```

Comment:
Isn't this something for in the release notes?

Reply:
is it ? I doubt anybody used that 'light_level_valid' value as it was true all the time.

Reply:
added a breaking change block to the PR description anyway.

Reply:
And added to that, is there any benefit of keeping light_level here?

Reply:
Why?